apiserver/leadership.go

```diff
@@ -73,8 +74,8 @@ type leadershipRevoker struct {
 }
 
 // RevokeLeadership is part of the leadership.Claimer interface.
-func (m leadershipRevoker) RevokeLeadership(applicationName, unitName string) error {
-	err := m.claimer.Revoke(applicationName, unitName)
+func (m leadershipRevoker) RevokeLeadership(applicationName string, unitName unit.Name) error {
+	err := m.claimer.Revoke(applicationName, unitName.String())
```

Comment:
**question:** `Revoke` cannot take a unitName too ?

Reply:
`Revoke` is too generic. 'Holders' can be things other than units.